Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         }
         
         eventDetailViewController.theEvent = event;
-        //eventDetailViewController.title = event.title;
```

Comment:
Personally I'd prefer to remove the second title rather than the first, but talk to the designers and do whatever they tell you. 

Alex

> On Apr 17, 2016, at 2:27 PM, AFrench <notifications@github.com> wrote:
> 
> In Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m:
> 
> > @@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
> >          }
> >          
> >          eventDetailViewController.theEvent = event;
> > -        //eventDetailViewController.title = event.title;
> No, the title wouldn't be set. I'm going to attach 2 pictures to show the difference, and you tell me which you would prefer:
> 
> 
> 
> 
> The only thing that changes is that on the top of the screen we have the title of the event show up twice, as opposed to having the title once and the view controller's title as "Details"
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>